tests/TestHarness/queries.py

```diff
@@ -346,7 +346,7 @@ def getEosAccount(self, name, exitOnError=False, returnType=ReturnType.json):
         return self.processCleosCmd(cmd, cmdDesc, silentErrors=False, exitOnError=exitOnError, exitMsg=msg, returnType=returnType)
 
     def getTable(self, contract, scope, table, exitOnError=False):
-        cmdDesc = "get table"
+        cmdDesc = "get table --time-limit 999"
```

Comment:
This will probably break 2.0 compatibility for performance test. This time limit is handled in `configure_version()` but was lost due to the splitting of Node.py into queries.

https://github.com/AntelopeIO/leap/pull/874/files#diff-4d1412150dda54fbc6280cc9053914a45d01e43085e35b0a5a68e9c8758ed42bR350 restores this in a way that won't break 2.0 compatibility.

Reply:
I see, that was brought from un-merged to main PR with Kevin's commit, should I just remove this change from this commit?  It has nothing to do with said python test

Reply:
removed it from this pr, so it properly has only one file with my changes

Reply:
Thanks